### PR TITLE
Adding additional tags to workflows

### PR DESCRIPTION
**Pull request by roiklorin**
added additional tags variables to workflows that deploy AWS resources

### CODE REVIEW

**Review by eytannnaim — APPROVED** [review]

File: .github/workflows/dsf_poc_cli.yml [hunk 1]
```diff
@@ -9,6 +9,11 @@ on:
       explicit_ref:
         required: true
         type: string
+      #additional_tags:
```
Comment (lindanasredin) [inline .github/workflows/dsf_poc_cli.yml]: Remove comment

File: .github/workflows/dsf_poc_cli.yml [hunk 2]
```diff
@@ -31,6 +38,10 @@ on:
         type: boolean
         default: false
         required: false
+      #additional_tags:
```
Comment (lindanasredin) [inline .github/workflows/dsf_poc_cli.yml]: Remove comment

File: .github/workflows/dsf_poc_cli.yml [hunk 3]
```diff
@@ -205,14 +217,14 @@ jobs:
     - name: Terraform Plan
       run: |
         terraform -chdir=$EXAMPLE_DIR workspace list
-        terraform -chdir=$EXAMPLE_DIR plan -var dam_license=license.mprv
+        terraform -chdir=$EXAMPLE_DIR plan -var dam_license=license.mprv -var additional_tags='${{ env.TF_VAR_additional_tags }}'
```
Comment (lindanasredin) [inline .github/workflows/dsf_poc_cli.yml]: No need to pass environment variables

File: .github/workflows/dsf_poc_cli.yml [hunk 4]
```diff
@@ -205,14 +217,14 @@ jobs:
...
 
       # On push to "main", build or change infrastructure according to Terraform configuration files
       # Note: It is recommended to set up a required "strict" status check in your repository for "Terraform Cloud". See the documentation on "strict" required status checks for more information: https://help.github.com/en/github/administering-a-repository/types-of-required-status-checks
     - name: Terraform Apply
       id: apply
       # if: github.ref == 'refs/heads/"master"' && github.event_name == 'push' || github.event_name == 'workflow_dispatch'
-      run: terraform -chdir=$EXAMPLE_DIR apply -var dam_license=license.mprv -auto-approve
+      run: terraform -chdir=$EXAMPLE_DIR apply -var dam_license=license.mprv -var additional_tags='${{ env.TF_VAR_additional_tags }}' -auto-approve
```
Comment (lindanasredin) [inline .github/workflows/dsf_poc_cli.yml]: Remove

File: .github/workflows/nightly_manager.yml [hunk 5]
```diff
@@ -12,8 +12,8 @@ jobs:
   master_basic_poc:
     uses: imperva/dsfkit/.github/workflows/nightly_sonar_poc_basic_cli.yml@master
     with:
-      branch: master
-      workspace: simple_cli_master_nightly
+      branch: adding-additional-tags-to-workflows
```
Comment (lindanasredin) [inline .github/workflows/nightly_manager.yml]: Leftovers

**Review by lindanasredin — CHANGES_REQUESTED** [review]

**Review by lindanasredin — APPROVED** [review]

**Pull request merged**